jupyterhub/jupyterhub/overlays/trusted-ca-bundle-path/kustomization.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+commonLabels:
+  app.kubernetes.io/part-of: jupyterhub
+
+bases:
+  - ../../base
+resources:
+  - trusted-cabundle-configmap.yaml
```

Comment:
this is the new configmap containing the PEM cert file, auto-injected by openshift